en/components/grid_virtualization.md

```diff
@@ -29,14 +29,20 @@ Virtualization is automatically enabled if the size of the related grid dimensio
 
```

Comment:
@ChronosSF I guess we can remove this content as the new text explains it in details.